tests-beta/ios/MASVS-STORAGE/MASTG-TEST-0x55-1.md

```diff
@@ -0,0 +1,30 @@
+---
+platform: ios
+title: Sensitive Data Not Excluded From Keyboard Caching
+id: MASTG-TEST-0x55-1
+type: [static]
+weakness: MASWE-0053
+---
+
+## Overview
+
+This test checks whether the target app prevents the caching of sensitive information entered into text fields. The keyboard may suggest previously entered text when typing in any app on the device.
+
+The following attributes, if present, will prevent the caching mechanism for text inputs:
+
+- [`UITextAutocorrectionTypeNo`](https://developer.apple.com/documentation/uikit/uitextautocorrectiontype/uitextautocorrectiontypeno)
+- [`secureTextEntry`](https://developer.apple.com/documentation/uikit/uitextinputtraits/1624427-securetextentry)
```

Comment:
should we require both? We need to explain this better, either here or in the evaluation using some examples maybe.

Reply:
I clarified it in the last commit, please have a look